src/main/java/Storage.java

```diff
@@ -0,0 +1,87 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+class Storage {
+
+    public static ArrayList<Task> tasks = new ArrayList<Task>();
+    public static File file = new File("data/duke.txt");
+    public void createFile() throws IOException {
+        File dir = new File("data/");
+        if (!dir.exists()) {
+            dir.mkdir();
+        }
+
+        File savedTasks = file;
+        if (!savedTasks.exists()) {
+            savedTasks.createNewFile();
+        }
+    }
+    public void writeTasks( ) throws IOException {
```

Comment:
Did you accidentally add a while space here? Minor issue but feels a bit weird 😆

Reply:
Good spot! I think the white space may have been on accident.